### PR TITLE
Add html and css design model based on bootstrap 4

### PR DESCRIPTION
**Pull request by areguera**

### CODE REVIEW

**Comment by arrfab** [comment]
Looks good, but maybe we can (for now) still use the previous logo (instead of centos-logo-white.png) and also favicon.ico instead of favicon.png ?

**Comment by areguera** [comment]
@arrfab centos-logos-white.png is the same image we use in wiki.centos.org, just with a different name to describe it is for use over dark backgrounds. Note that we have design differences between the logo in wiki.centos.org and the logo in headers. Which one should we use?

I am preparing a new commit to remove favicon.png an use favicon.ico instead. Plus the html corrections to make it visible.

File: files/centos-design/header.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <!-- CentOS customization -->
+    <link rel="stylesheet" href="css/centos.css">
```
Comment (arrfab) [inline files/centos-design/header.html]: shouldn't it be /centos-design/css/centos.css ? all files would land in that path on httpd server, so should be referenced with full path

Comment (areguera) [reply files/centos-design/header.html]: Yes. I am preparing the fix.

Comment (arrfab) [reply files/centos-design/header.html]: Thanks, looks good

**Comment by arrfab** [comment]
Merging it in staging, but the footer.html will probably not be used, also with static caroussel with all sponsors (and it's always changing for centos.org itself, through different git repo) but at least that shows good ideas on what/how to use it to build a page with header + footer for dynamic content.
Thanks a lot !

**Pull request merged**